### PR TITLE
Add Dig, fix #80

### PR DESCRIPTION
**Pull request by MasterPi-2124**

### CODE REVIEW

**Comment by MasterPi-2124** [comment]
- Added DIG to assetlists
- Added DIG logo

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -919,6 +919,34 @@
...
+      "name": "Dig Chain",
+      "display": "dig",
+      "symbol": "DIG",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-128",
+        "source_denom": "udig"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/DIG.png"
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:948]: https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dig.png

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -919,6 +919,34 @@
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dsm.png",
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dsm.svg"
       }
+    },
+    {
+      "description": "The native staking and governance token of the Osmosis.",
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:924]: The native token of Dig Chain

Comment (MasterPi-2124) [reply osmosis-1/osmosis-1.assetlist.json]: I fixed it. Thanks for your help !!!

**Comment by IbrarMakaveli** [comment]
Can you also change images/DIG.png to lowercase : images/dig.png

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (MasterPi-2124) [reply osmosis-1/osmosis-1.assetlist.json]: I fixed it. Thanks for your help !!!

**Comment by MasterPi-2124** [comment]
Yes, i changed it in newest commit.

**Review by IbrarMakaveli — APPROVED** [review]

**Pull request merged**